packages/storybook/stories/ParticipantList/ParticipantList.stories.tsx

```diff
@@ -116,7 +118,7 @@ export const ParticipantList: () => JSX.Element = () => {
   };
 
   return (
-    <Stack>
+    <Stack style={{ height: '2rem' }}>
```

Comment:
What is this height ensuring? Ideally are components themselves handle awkward edge cases or min-heights